tick/base/array_test/src/timer.h

```diff
@@ -6,26 +6,38 @@
 //  Copyright (c) 2015 bacry. All rights reserved.
```

Comment:
Only in the performance test source. But it was easier to refactor than to make the macros conform to cpplint.